aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -1216,6 +1340,11 @@ module aptos_framework::stake {
                 rewards_amount,
             },
         );
+
+        active_amount
+            + pending_active_amount
+            + if (lockup_expired) { 0 } else { current_pending_inactive_amount }
+            + rewards_amount + txn_fee_amount
```

Comment:
probably leave a comment to prove that when commit == true, this value == stake_pool.active